spec/factories/training_slides.rb

```diff
@@ -19,5 +19,8 @@
 
 FactoryBot.define do
   factory :training_slide do
+    id { 456875}
+    title { 'How to create a slide' }
+    slug { 'how-to-create-a-slide' }
   end
 end
```

Comment:
Added this because I want to test the behavior for an orphan slide (one that doesn't belong to any module).